### ISSUE TITLE
Can't log into Exilence Next after auto log-out

### ISSUE THREAD

**Issue by Lexediros** [viktorgullmark/exilence-next#891]
I was automatically logged out of Exilence Next after not using it for a while (I just started using again after 1-2 leagues ago, so it's been quite some time). However, after logging in and clicking "Authorize", it brings me to an error page as shown below.

The link that has the error is : https://next.exilence.app/api/authentication/redirect?code=253f87a80416e25003d8c5e800a38139c5b76794&state=f1e411a4-c8f1-4c96-94a5-445b4e01a683

I have also tried using the link given in the login screen of exilence next : https://www.pathofexile.com/oauth/authorize?client_id=exilence&response_type=code&scope=account:stashes account:profile account:characters&state=f1e411a4-c8f1-4c96-94a5-445b4e01a683&redirect_uri=https://next.exilence.app/api/authentication/redirect

but that also doesn't work. Is there any fix to this? Is there an update I have to download for it to work again? 

![image](https://github.com/viktorgullmark/exilence-next/assets/54137280/5cefc9a2-048f-4d23-90fd-0f855a3bab04)
![image](https://github.com/viktorgullmark/exilence-next/assets/54137280/c00ae631-4a46-48ee-aa36-1d794d0549d8)
![image](https://github.com/viktorgullmark/exilence-next/assets/54137280/7c71311a-ba54-4569-ae04-dda87a24998d)

**Comment by Stanley-Yao** [comment]
same here, I think the website dns record is expired, maybe the author discontinued the web service :(

**Comment by devwithilja** [comment]
Yeah I run into the same issue, but according to some other posts this service was maybe stopped from the owners and there is now a comunity fork of it

https://github.com/exilence-ce
